lib/features/email/data/datasource_impl/email_hive_cache_datasource_impl.dart

```diff
@@ -149,4 +152,37 @@ class EmailHiveCacheDataSourceImpl extends EmailDataSource {
       return await _emailCacheManager.storeEmail(accountId, session.username, email.toEmailCache());
     }).catchError(_exceptionThrower.throwException);
   }
+
+  @override
+  Future<void> storeOpenedEmailToCache(Session session, AccountId accountId, DetailedEmail detailedEmail) {
+    return Future.sync(() async {
+      final task = HiveTask(
+          runnable: () async {
+
+            final isDuplicateDetailedEmail = await _detailedEmailCacheManager.checkDuplicateOpenedEmail(accountId, session.username,detailedEmail);
```

Comment:
```suggestion
            final detailedEmailExisted = await _detailedEmailCacheManager.checkDuplicateOpenedEmail(accountId, session.username,detailedEmail);
```

Reply:
And I think we need to change the name `checkDuplicateOpenedEmail` also.
How about `openedDetailsEmailCacheHit`